packages/apollo-cli/src/commands/assembly/add-fasta.ts

```diff
@@ -43,25 +36,32 @@ export default class Get extends BaseCommand<typeof Get> {
     }),
     index: Flags.string({
       char: 'x',
-      description:
-        'URL of the index. Required if input is an external source and ignored if input is a local file',
+      description: 'URL of the index. Required if input is an external source',
     }),
     force: Flags.boolean({
       char: 'f',
       description: 'Delete existing assembly, if it exists',
     }),
+    'no-db': Flags.boolean({
+      char: 'n',
+      description: wrapLines(
+        "Do not load the fasta sequence into the Apollo database. \
+      This option assumes the fasta file is bgzip'd with `bgzip` and indexed with `samtools faidx`.\
+      Indexes should be named <my.fasta.gz>.gzi and <my.fasta.gz>.fai",
```

Comment:
Auto-detection is good, but there should also be a way to explicitly pass the fai and gzi files.